lib/decoding_text_effect.dart

```diff
@@ -226,11 +226,17 @@ class _DecodingTextEffectState extends State<DecodingTextEffect> {
     _startDecoding();
   }
 
+  @override
+  void dispose() {
+    super.dispose();
+    _timer?.cancel();
+  }
+
   void _startDecoding() async {
     if (_effect == DecodeEffect.fromStart) {
       int _index = 0;
       int _count = 0;
-      Timer.periodic(_refreshDuration, (timer) {
+      _timer = Timer.periodic(_refreshDuration, (timer) {
```

Comment:
Great work, @bigzhu .
Reuesting a small change.
It would be better if you initialized `_timer` for all the 5 `DecodeEffects`.

Reply:
already added